lib/lightning/work_orders.ex

```diff
@@ -296,33 +350,30 @@ defmodule Lightning.WorkOrders do
   def retry_many([%WorkOrder{} | _rest] = workorders, opts) do
     orders_ids = Enum.map(workorders, & &1.id)
 
-    attempt_run_numbers_query =
-      from(ar in AttemptRun,
-        join: att in assoc(ar, :attempt),
-        join: r in assoc(ar, :run),
+    attempt_numbers_query =
+      from(att in Attempt,
         where: att.work_order_id in ^orders_ids,
         select: %{
-          id: ar.id,
+          id: att.id,
           row_num:
             row_number()
             |> over(
               partition_by: att.work_order_id,
-              order_by: coalesce(r.started_at, r.inserted_at)
+              order_by: coalesce(att.started_at, att.inserted_at)
             )
```

Comment:
Do we still need to partition/create row numbers?

Actually I'm not sure I understand why we need to find any attempts - if we're only given a workorder; can we not create a new attempt based on the workorder?